### PR TITLE
Fixes `sbt assembly` usage in docs and travis.

### PR DESCRIPTION
**Pull request by janisz**
Fixes: MARATHON-7269 (#5370)
Refs: D730

### CODE REVIEW

**Comment by janisz** [comment]
Ping

**Comment by janisz** [comment]
Ping

**Comment by janisz** [comment]
Ping

**Comment by janisz** [comment]
@unterstein Can you take a look at this?

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.sbt
     - $HOME/.ivy2
 script:
-  - sbt "set parallelExecution in Test := false" clean coverage doc assembly
+  - sbt clean scapegoat doc coverage package
```
Comment (jeschkies) [inline .travis.yml]: Hm, does this run the unit tests?

Comment (janisz) [inline .travis.yml]: Good point. I'll fix it

Comment (janisz) [inline .travis.yml]: Fixed

**Comment by janisz** [comment]
ping!

File: .travis.yml (hunk 1 above)
Comment (jeschkies) [inline .travis.yml]: I think this should be `sbt clean scapegoat doc coverage test package`

Comment (janisz) [inline .travis.yml]: My mistake. Fixed

Comment (janisz) [inline .travis.yml]: But now test fails due to low coverage https://travis-ci.org/janisz/marathon/builds/228659037

```
[info] Statement coverage.: 21.22%
[info] Branch coverage...: 19.14%
[info] Coverage reports completed
[error] Coverage is below minimum [21.22% < 75.0%]
```

Comment (jeschkies) [inline .travis.yml]: Ok. That is strange. Let me look this up.

Comment (jeschkies) [inline .travis.yml]: `sbt clean scapegoat doc coverage test` works fine on master. Please keep in mind that `coverage` inserts instructions into the code. When you package for deployment, you should clean before: `sbt clean package`.

Comment (jeschkies) [inline .travis.yml]: `sbt clean scapegoat doc coverage test coverageReport` yields this error

```
[info] Statement coverage.: 73.42%
[info] Branch coverage....: 70.45%
[info] Coverage reports completed
[error] Coverage is below minimum [73.42% < 75.0%]
```

Could lower the coverage in this diff to 70%? Thanks a ton.

File: build.sbt [hunk 2]
```diff
@@ -125,7 +125,7 @@ lazy val commonSettings = testSettings ++
 
   scapegoatVersion := "1.3.0",
 
-  coverageMinimum := 75,
+  coverageMinimum := 70,
```
Comment (janisz) [inline build.sbt]: :disappointed:

**Comment by janisz** [comment]
Still failing https://travis-ci.org/janisz/marathon/builds/254749599#L2136

**Comment by jeschkies** [comment]
Thanks @janisz!

**Pull request merged**